msvc/sdk-build.targets

```diff
@@ -64,7 +64,15 @@
       <PackageFile Include="@(_WinMDFileNames->Distinct())" Condition="Exists('%(FullPath)')">
         <PackagePath>$(PackageContentBasePath)\%(Filename)%(Extension)</PackagePath>
       </PackageFile>
-    </ItemGroup>
+      
+      <!-- Package up .pri files associated with any .winmds -->
+      <_PriFileNames Include="$(ProjectPriFullPath)">
+      </_PriFileNames>
+      <PackageFile Include="@(_PriFileNames)" Condition="Exists('%(RootDir)%(Directory)%(Filename).winmd')">
```

Comment:
> Condition="Exists('%(RootDir)%(Directory)%(Filename).winmd') [](start = 45, length = 61)

This condition really isn't what we want.

You should be checking for the existence of the .pri file not the .winmd and then And ing with the notion that "this project is going to produce a winmd". 

Are there any scenarios where a pri file is needed but a winmd isn't created?